templates/components/_nav_desktop.html.twig

```diff
@@ -0,0 +1,214 @@
+<nav class="gtm__navigation__desktop l__wrapper l__row l__row--h-stretch l__hide--on-small-laptop {{ app.user ? '' : 'space--20-0' }}">
+    <ul class="list__links list__links--row list__links--no-decor list__links--nav">
+        {% if app.user %}
```

Comment:
`has_role_adherent`

Reply:
nope, a user doesnt have to adhere anymore \o/

Reply:
`has_role_user` then :p